src/affine.jl

```diff
@@ -1,79 +1,149 @@
-#TODO more documentation?
+"""
+    tfm = linmap(mat, img, initial_tfm=IdentityTransformation())
+
+Construct a linear transformation from the values in `mat`.
+`mat` can be any list of `N^2` values, where `N` is the dimensionality.
+`img` is supplied solely to get the dimensionality, so that the size of the matrix in `tfm`
+is inferrable.
```

Comment:
The inferrabiliy is more about how the function works than how to use the function, so maybe it should go into the function as a comment instead?